mars/metric/backends/metric.py

```diff
@@ -0,0 +1,163 @@
+# Copyright 1999-2021 Alibaba Group Holding Ltd.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import time
+
+from abc import ABC, abstractmethod
+from threading import Lock
+from typing import Dict, Optional, Tuple
+
+_THRESHOLD = 2000
+_RECORDED_INTERVAL_SECS = 1
+
+
+class _MutexValue(object):
```

Comment:
There is no need to inherit from object.

Reply:
Good suggestion. Thanks.